### ISSUE TITLE
Removing viewBox

### ISSUE THREAD

**Issue by lexeii** [RazrFalcon/svgcleaner#196]
Recently, I have been processing hundreds and hundreds of SVG icon files created in Inkscape. In all of them, deleting a SVG attribute `viewBox` is possible because it duplicates existing attributes `width` and `height`.

In:
``` xml
<svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
```

Out:
``` xml
<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg">
```

In rare cases (which I have not seen yet), in order to implement a more general logic, you can remove an attribute only when it satisfies the condition:

``` 
viewBox="0 0 width height"
```

**Comment by RazrFalcon** [comment]
It's actually a svgcleaner bug. It should not remove `width`, `height` or `viewBox`. This feature will be removed in later releases.

**Comment by lexeii** [comment]
Oh, perhaps I was misunderstood. It doesn’t delete the `viewBox` attribute right now, but I would like it to delete. Now I do it every time manually, but sometimes I forget to do it.

**Comment by RazrFalcon** [comment]
I've checked and yes, svgcleaner does not remove `viewBox`. And this is not optional. Probably older versions did.

And not, this will not be added, because width/height and viewBox have different meaning. You can't simple remove one of them.

**Comment by lexeii** [comment]
I agree — in the general case — yes. But more often it's just duplication.
OK, you may close the issue.

**Comment by RazrFalcon** [comment]
Yes, in you particular case, `viewBox` can be omitted. And the algorithm should also check `preserveAspectRatio` attribute.

I will leave this open, but i'm not sure that svgcleaner will receive an update anytime soon.

**Comment by lexeii** [comment]
I understand that there will be no updates in the near future, this can be seen in your latest answers. Perhaps I will add two more issues. For the future. I hope that someday you will have time and desire to continue to work on improvements.

Thank you!
